e2e_samples/dataset_versioning/infra/setup.sh

```diff
@@ -9,20 +15,20 @@ while true; do
     echo "    Keyvault: kv-terraform-prime"
     echo "    Service Principal: sp-resourcegroup-Prime-tf"
     echo -e "\n"
-    read -p "Enter value: " RESOURCE_GROUP_NAME TF_STATE_STORAGE_ACCOUNT_NAME KEYVAULT_NAME LOCATION
+    read -rp "Enter value: " RESOURCE_GROUP_NAME TF_STATE_STORAGE_ACCOUNT_NAME KEYVAULT_NAME LOCATION
 
     SP_NAME="sp-${RESOURCE_GROUP_NAME}-tf"
     TF_STATE_CONTAINER_NAME=terraform-state
 
     # Create the resource group
-    if [ $(az group exists --name $RESOURCE_GROUP_NAME) = false ]; then
+    if [ "$(az group exists --name "$RESOURCE_GROUP_NAME" --output tsv)" = false ]; then
```

Comment:
Is this better practice to have double quotation around argument in bash?  Hopefully there was auto correct on this in vscode.